core/src/main/java/org/apache/struts2/interceptor/DefaultResourceIsolationPolicy.java

```diff
@@ -0,0 +1,49 @@
+package org.apache.struts2.interceptor;
+
+import org.apache.logging.log4j.util.Strings;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ *
+ * Default resource isolation policy used in {@link FetchMetadataInterceptor} that
+ * implements the {@link ResourceIsolationPolicy} interface. This default policy is based on
+ * <a href="https://web.dev/fetch-metadata/">https://web.dev/fetch-metadata/</a>.
+ *
+ * @see <a href="https://web.dev/fetch-metadata/">https://web.dev/fetch-metadata/</a>
+ *
+ * @author Santiago Diaz - saldiaz@google.com
+ * @author Giannis Chatziveroglou - giannic@google.com
```

Comment:
Please remove these tags and emails, after merging this code it will be owned by ASF and you want to be bothered by users to help them resolve their problems, it's community duty :)

http://tinyurl.com/mw7t6

Reply:
haha that's fair! :)